### PR TITLE
feat(EMS-1715-2194): IaC, update JWT expiry time

### PR DESCRIPTION
**Pull request by abhi-markan**
## Introduction ✏️ 
Creates various Azure resources such as resource groups, app service plans, log analytics workspaces, container registries, virtual networks, subnets, VNET peerings, web apps, front doors, private endpoints, DNS configurations and diagnostic settings.

It also sets environment variables and configures the web apps with app settings, logging options and alerts.

## Resolution ✔️ 
* IaC upgrade.
* Slot creation using `--deployment-container-image-name` for first deployment.

## Miscellaneous ➕ 
* Dependencies updates.
* Documentation updates.
* Formatting updates.
* Updated `actions/setup-node` to `v4`.
* Regenerated Keystone config.
* Updated JWT expiry to `12h` for consistency.
* Removed JWT `iat` property from payload since it will be auto-populated by JWT.

### CODE REVIEW

File: src/api/constants/index.ts [hunk 1]
```diff
@@ -116,7 +116,7 @@ export const ACCOUNT = {
       STRING_ENCODING: 'ascii' as BufferEncoding,
     },
     TOKEN: {
-      EXPIRY: '8h',
+      EXPIRY: '12h',
```
Comment (ttbarnes) [inline src/api/constants/index.ts:119]: Do we need a ticket for this?

Comment (abhi-markan) [reply src/api/constants/index.ts:119]: Yes please

Comment (ttbarnes) [reply src/api/constants/index.ts:119]: Have created a task here and added updated the PR title https://ukef-dtfs.atlassian.net/browse/EMS-2194

**Comment by ttbarnes** [comment]
@abhi-markan FYI there are currently some flaky E2E tests after companies house changes. Re-running "failed jobs" should do the trick. Need to investigate this, unable to replicate locally, need to mimic GHA environment

**Review by Zainzzkk — DISMISSED** [review]

**Review by Zainzzkk — APPROVED** [review]

**Review by ttbarnes — APPROVED** [review]

**Pull request merged**